components/x11/redshift/redshift.p5m

```diff
@@ -0,0 +1,77 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Sergey Avseyev
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/$(MACH64)/redshift
+file path=usr/bin/$(MACH64)/redshift-gtk
+file path=usr/bin/redshift
+file path=usr/bin/redshift-gtk
+file path=usr/lib/python3.4/site-packages/redshift_gtk/__init__.py
+file path=usr/lib/python3.4/site-packages/redshift_gtk/__pycache__/__init__.cpython-34.pyc
```

Comment:
Please. move python files to vendor-packages. Also */__pycached__/* files shouldn't be mentioned, transforms/autopyc should take care of this.